Implement linear regression

drmarjanovic opened the pull request:
Summary:

- Implement linear regression
- Short description added in `description.txt`
- **Outliers** removed manually from `data\test.csv`

Need to implement:

- automatically removing **outliers** from provided dataset

drmarjanovic committed:
Implement linear regression

stkeky commented on ex-01-linear-regression/test.py:
```diff
@@ -0,0 +1,39 @@
+
+    # Normalize input (X) and preprocessed output (Y)
+    X_norm = normalize(X)
+    Y_norm = normalize(preprocess(Y))
```
Can't you use here `Y_prep` instead of preprocessing `Y` again?

drmarjanovic replied:
Yes, we can use that. 😄

drmarjanovic committed:
Simplify normalizing output with already preprocess

bojanbla committed:
Describe problem with alternative RANSAC algorithm

stkeky committed:
Remove outliers and change description

Removed 9 outliers, written task specific decription and changed mean to
median in normalization.

stkeky committed:
Removed unused imports

stkeky committed:
Update README.md

bojanbla committed:
Remove unused methods
